examples/se2_localization_ukfm.cpp

```diff
@@ -0,0 +1,426 @@
+/**
+ * \file se2_localization.cpp
+ *
+ *  Created on: Dec 10, 2018
+ *     \author: jsola
+ *
+ *  ---------------------------------------------------------
+ *  This file is:
+ *  (c) 2018 Joan Sola @ IRI-CSIC, Barcelona, Catalonia
+ *
+ *  This file is part of `manif`, a C++ template-only library
+ *  for Lie theory targeted at estimation for robotics.
+ *  Manif is:
+ *  (c) 2018 Jeremie Deray @ IRI-UPC, Barcelona
+ *  ---------------------------------------------------------
+ *
+ *  ---------------------------------------------------------
```

Comment:
Fix authorship and doc details throughout this comments section

Reply:
I'll fix all the doc once the code itself is :+1: